src/raftstore/store/peer.rs

```diff
@@ -564,10 +598,9 @@ impl Peer {
         send_to_quorum_ts + self.cfg.raft_store_max_leader_lease
     }
 
-    fn update_leader_lease(&mut self, ready: &Ready) {
+    fn on_role_changed(&mut self, ready: &Ready) {
         // Update leader lease when the Raft state changes.
```

Comment:
The comments should be updated when the function changes with its name modified.